test/problems/second_quantization/vibrational/test_vibrational_problem.py

```diff
@@ -24,8 +24,10 @@ class TestVibrationalProblem(QiskitNatureTestCase):
     def test_second_q_ops_without_transformers(self):
         """Tests that the list of second quantized operators is created if no transformers
         provided."""
-        expected_num_of_sec_quant_ops = 1
-        logfile = self.get_resource_path('CO2_freq_B3LYP_ccpVDZ.log')
+        expected_num_of_sec_quant_ops = 5
```

Comment:
FYI: this number changed because we added the missing `aux_ops` to the `VibrationalProblem`